<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Donation</class>
 <widget class="QMainWindow" name="Donation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/imgs/logo-removebg-preview_50_1_50.png</pixmap>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>240</y>
      <width>808</width>
      <height>243</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Are you 16 â€“ 65 years old ?&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Do you currently weigh less than 50kg ?&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Do you have any terminal illness ?&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string/>
           </property>
           <widget class="QRadioButton" name="radioButton">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>30</y>
              <width>127</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>Yes</string>
            </property>
           </widget>
           <widget class="QRadioButton" name="radioButton_2">
            <property name="geometry">
             <rect>
              <x>140</x>
              <y>30</y>
              <width>194</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>No</string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string/>
           </property>
           <widget class="QRadioButton" name="radioButton_3">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>30</y>
              <width>127</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>Yes</string>
            </property>
           </widget>
           <widget class="QRadioButton" name="radioButton_4">
            <property name="geometry">
             <rect>
              <x>140</x>
              <y>30</y>
              <width>194</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>No</string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string/>
           </property>
           <widget class="QRadioButton" name="radioButton_5">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>30</y>
              <width>127</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>Yes</string>
            </property>
           </widget>
           <widget class="QRadioButton" name="radioButton_6">
            <property name="geometry">
             <rect>
              <x>140</x>
              <y>30</y>
              <width>194</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>No</string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>500</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(166, 166, 166);
text-color:rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>651</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Where would you like to donate:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">
border-weight:2;
</string>
       </property>
       <item>
        <property name="text">
         <string>CairoCentralBloodBank</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Alexandria,BakosBloodBank</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Al-Gharbia,TantaBloodBank</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Al-Beheira, Damnhour Blood Bank</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Qena Blood Bank</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Luxor International Hospital</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>New valley, El Kharga General Hospital</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Al salam portsaid Hospital</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>General Suez Hospital</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
